js/ui.js

```diff
@@ -13,8 +13,14 @@ function display(data) {
 		if (! target)
 			return
 
-		if (typeof value == 'number')
-			value = String(value.toFixed(2)).replace('.', ',')
+		if (typeof value == 'number') {
+			if (target.hasAttribute('data-round'))
+				value = Math.round(value)
+			else
+				value = value.toFixed(2)
```

Comment:
Wouldn't this be simpler with a ternary: `value = value.toFixed(target.hasAttribute('data-round') ? 0 : 2)`.